src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -144,7 +219,7 @@ private void RemoveAllCorrelationIdentifierContexts()
             }
         }
 
-        private void SetLogContext(string service, string version, string env, ulong traceId, ulong spanId)
+        private void SetLogContext(ulong traceId, ulong spanId)
```

Comment:
Additional unrelated change I made while I was here: The service, version, and env of the Tracer never change, so I removed those argument from the method.